libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -260,93 +278,41 @@ class MapboxNavigationTest {
     }
 
     @Test
-    fun reRoute_called() {
-        val offRouteObserverSlot = slot<OffRouteObserver>()
-        verify { tripSession.registerOffRouteObserver(capture(offRouteObserverSlot)) }
-
-        offRouteObserverSlot.captured.onOffRouteStateChanged(true)
-
-        verify(exactly = 1) { directionsSession.requestRoutes(any(), any()) }
-    }
-
-    @Test
-    fun getEnhancedLocation_reRoute() {
```

Comment:
Are we adding these 👇 somewhere else? Want to make sure that we're not removing coverage.

Reply:
this logic was moved to RouteOptionsProvider

Reply:
And I imagine the tests as well, right? That's what I wanted to confirm 😅 Let's make sure we drop a note in https://github.com/mapbox/mapbox-navigation-android/issues/3236 if not (based on https://github.com/mapbox/mapbox-navigation-android/pull/3184/files#r445583452 we're not covering all scenarios as part of this PR).

Reply:
yeah, nice note. This logic was moved to `MapboxRerouteController`. Added tests